docker/scripts/dev_start.sh

```diff
@@ -68,7 +69,8 @@ DOCKER_MEMORY="${DOCKER_MEMORY:-8g}"
 DOCKER_IMAGE_REPO=${DOCKER_IMAGE_REPO:="wheelos/apollo"}
 DOCKER_IMAGE_TAG_X86_64=${DOCKER_IMAGE_TAG_X86_64:="dev-x86_64-20.04-20250713_1555"}
 DOCKER_IMAGE_TAG_X86_64_TESTING=${DOCKER_IMAGE_TAG_X86_64_TESTING:="dev-x86_64-20.04-20250710_2109"}
-DOCKER_IMAGE_TAG_AARCH64=${DOCKER_IMAGE_TAG_AARCH64:="dev-aarch64-22.04-20251107_1200"}
+[[ "$RELEASE_TARGET" == "20.04" ]] && DOCKER_IMAGE_TAG_AARCH64=${DOCKER_IMAGE_TAG_AARCH64:="dev-aarch64-20.04-20250714_2123"}
```

Comment:
The selection depends on the operating system and chip; a refactoring is required.